src/modules/form/components/DynamicForm.tsx

```diff
@@ -91,25 +90,14 @@ export const DynamicFormWithoutHandlers = forwardRef<
       ref,
       () => ({
         SubmitForm: () => {
-          // onSubmit({ values: getCleanedValues(), confirmed: false, });
           onSubmit({ ...getValuesForSubmit(), confirmed: false });
         },
-        SaveIfDirty: () => {
-          if (!handlers.methods.formState.isDirty || props.locked) return;
```

Comment:
This was causing a bug that was caught in system tests.
To reproduce the issue:
* Open a household intake view for a new household (all wip members)
* Enter some responses onto the first intake
* Hit `Next Client`
  * Expected: performs a save with a horizontal loading bar, then proceeds to next tab when complete
  * Bug: does not perform a save, does not proceed to next tab